src/containers/Node/NodeStructure/Pdisk.tsx

```diff
@@ -54,32 +55,36 @@ const vDiskTableColumnsNames: Record<VDiskTableColumnsIdsValues, string> = {
 function getColumns({
     pDiskId,
     selectedVdiskId,
-    nodeHref,
+    nodeId,
 }: {
     pDiskId: number | undefined;
     selectedVdiskId?: string;
-    nodeHref?: string | null;
+    nodeId?: string | number;
 }) {
     const columns: Column<PreparedStructureVDisk>[] = [
         {
             name: VDiskTableColumnsIds.slotId,
             header: vDiskTableColumnsNames[VDiskTableColumnsIds.slotId],
             width: 100,
             render: ({row}) => {
-                let vdiskInternalViewerLink = '';
+                const vDiskSlotId = row.VDiskSlotId;
+                let vdiskInternalViewerLink = null;
 
-                if (nodeHref && pDiskId !== undefined && row.VDiskSlotId !== undefined) {
-                    vdiskInternalViewerLink +=
-                        nodeHref +
-                        'actors/vdisks/vdisk' +
-                        pad9(pDiskId) +
-                        '_' +
-                        pad9(row.VDiskSlotId);
+                if (
+                    valueIsDefined(nodeId) &&
+                    valueIsDefined(pDiskId) &&
+                    valueIsDefined(vDiskSlotId)
+                ) {
+                    vdiskInternalViewerLink = createVDiskDeveloperUILink({
+                        nodeId,
+                        pDiskId,
+                        vDiskSlotId,
+                    });
                 }
 
                 return (
                     <div className={b('vdisk-id', {selected: row.id === selectedVdiskId})}>
-                        <span>{row.VDiskSlotId}</span>
+                        <span>{vDiskSlotId}</span>
                         {vdiskInternalViewerLink && (
                             <Button
```

Comment:
Button here and in VDisk are without text. Lets add title to provide more accessibility.